webapp/graphql/User.js

```diff
@@ -258,3 +258,24 @@ export const checkSlugAvailableQuery = gql`
     }
   }
 `
+
+export const currentUserQuery = gql`
+  ${userFragment}
+  query {
+    currentUser {
+      ...user
+      email
+      role
+      about
+      locationName
+      locale
+      allowEmbedIframes
```

Comment:
I miss  `contributionsCount` and `commentedCount`

Reply:
That's on purpose. Fetching the two will cause unnecessary N+1 queries to the database. It seems that we don't use the counts in the frontend.

Reply:
And the profile page?

Reply:
see webapp/pages/profile/_id/_slug.vue line 179 and 191
`<hc-count-to slot="count" :end-val="user.contributionsCount" />`
`<hc-count-to slot="count" :end-val="user.commentedCount" />`

Reply:
The `user` is not coming from `store/auth` but from a separate graphql query: https://github.com/Human-Connection/Human-Connection/blob/master/webapp/pages/profile/_id/_slug.vue#L507

So that's no problem. The `currentUser` is what you see when you go to your avatar menu and update e.g. your social media.

Reply:
OK. Got it. In the profile we call the UserQuery where the counts are included. So you are right. It is not needed here.